swap/htlc.go

```diff
@@ -32,6 +32,9 @@ type ScriptVersion uint8
 const (
 	// HtlcV1 refers to the original version of the HTLC script.
 	HtlcV1 ScriptVersion = iota
+
+	// HtlcV2 refers to the improved or V2 version of the HTLC script.
```

Comment:
sorry to keep nitting thing. "improved or V2" sounds like this represents two different things to me. 
My previous comment was more to briefly describe how the script is different form V1, but non-blocking.